cvat-ui/src/components/create-task-page/create-task-content.tsx

```diff
@@ -54,6 +54,7 @@ const defaultState = {
         lfs: false,
         useZipChunks: true,
         useCache: true,
+        sortingMethod: SortingMethods.DEFAULT,
```

Comment:
Better to rename all methods to something meaningful. For example, it is difficult understand what DEFAULT means. But lexicographical is much better.